### PR TITLE
IMAGINGCLOUD-88 - Aspose.Imaging v18.6 release integration

### PR DESCRIPTION
**Pull request by MaxCrank**
Removed overkill test cases after moving them to the lower layer; updated README; updated POM; update code to ensure JDK 1.6 support; replaced JDK 1.8 with JDK 1.6 and Maven with 3.2.5, moved them to private repo; removed connection timeout handling since it's infinite by default; added temp folder usage for tests.

### CODE REVIEW

File: src/test/java/com/aspose/imaging/cloud/test/base/ApiTester.java [hunk 1]
```diff
@@ -53,99 +55,136 @@
...
         "bmp",
-        "jpg",
-        "psd",
-        "tiff",
         "gif",
+        "jpg",
         "png",
-        "j2k",
-        "webp"
+        "psd",
```
Comment (roxboris) [inline src/test/java/com/aspose/imaging/cloud/test/base/ApiTester.java]: @MaxCrank Are you sure that need to remove j2k, webp formats from tests?

File: src/test/java/com/aspose/imaging/cloud/test/api/UpdateImageApiTests.java [hunk 2]
```diff
@@ -67,36 +62,8 @@ public void setUp() throws Exception {
      */
     @Test
 	@Parameters({
-		".bmp, true,",
```
Comment (roxboris) [inline src/test/java/com/aspose/imaging/cloud/test/api/UpdateImageApiTests.java]: @MaxCrank Why you remove these test cases?

File: src/main/java/com/aspose/imaging/cloud/sdk/invoker/internal/ApiInvoker.java [hunk 3]
```diff
@@ -78,7 +78,7 @@
      */
     public ApiInvoker(IRequestHandler[] requestHandlers)
     {
-        String sdkVersion = "18.4";
+        String sdkVersion = "18.6";
```
Comment (roxboris) [inline src/main/java/com/aspose/imaging/cloud/sdk/invoker/internal/ApiInvoker.java]: @MaxCrank I think it makes sense to move the version to the common constant.

File: src/test/java/com/aspose/imaging/cloud/test/api/UpdateImageApiTests.java (hunk 2 above)
Comment (MaxCrank) [reply src/test/java/com/aspose/imaging/cloud/test/api/UpdateImageApiTests.java]: @roxboris please check corporate mail regarding tests across all levels.

File: src/test/java/com/aspose/imaging/cloud/test/base/ApiTester.java (hunk 1 above)
Comment (MaxCrank) [reply src/test/java/com/aspose/imaging/cloud/test/base/ApiTester.java]: @roxboris please check corporate mail regarding tests across all levels.

File: src/main/java/com/aspose/imaging/cloud/sdk/invoker/internal/ApiInvoker.java (hunk 3 above)
Comment (MaxCrank) [reply src/main/java/com/aspose/imaging/cloud/sdk/invoker/internal/ApiInvoker.java]: @roxboris you're right, fixed.

**Pull request merged**